src/chapters/configuring.xml

```diff
@@ -2772,6 +2794,24 @@ ed272546-87bd-4db9-acba-e36e1a9ca20a
                                         session will use the user's default shell.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>server-alive-interval</parameter></entry>
```

Comment:
I'm not sure it makes sense to include the keepalive interval within a new "environment settings" section. Since the keepalive interval deals with the SSH connection itself, what about the existing "network parameters" section?

http://guacamole.incubator.apache.org/doc/gug/configuring-guacamole.html#ssh-network-parameters

IMHO, the keepalive interval is by definition a network parameter.